apps/desktop/scripts/MacApiBridge/build.sh

```
@@ -0,0 +1,382 @@
+    security create-keychain -p "$TEMP_KEYCHAIN_PASSWORD" "$TEMP_KEYCHAIN"
+    security set-keychain-settings -lut 21600 "$TEMP_KEYCHAIN"
+    security unlock-keychain -p "$TEMP_KEYCHAIN_PASSWORD" "$TEMP_KEYCHAIN"
+
+    echo "  Importing certificate..."
+    security import "$P12_PATH" -k "$TEMP_KEYCHAIN" -P "$CSC_KEY_PASSWORD" -T /usr/bin/codesign -T /usr/bin/security
+
+    # Set partition list to allow codesigning without prompting
+    security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k "$TEMP_KEYCHAIN_PASSWORD" "$TEMP_KEYCHAIN"
+
+    # Add to search list
+    security list-keychains -d user -s "$TEMP_KEYCHAIN" $(security list-keychains -d user | sed s/\"//g)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)